v0/destinations/marketo_bulk_upload/fileUpload.js

```diff
@@ -29,14 +62,15 @@ const getHeaderFields = config => {
   return Object.keys(columnField);
 };
 
-const getFileData = (inputEvents, config) => {
+const getFileData = async (inputEvents, config) => {
   const input = inputEvents;
   const messageArr = [];
   let startTime;
   let endTime;
   let requestTime;
   startTime = Date.now();
-  const headerArr = getHeaderFields(config);
+  const fieldArrNames = await fieldSchema(config);
+  const headerArr = getHeaderFields(config, fieldArrNames);
```

Comment:
As `fieldArrNames` is only being passed to `getHeaderFields`, you can invoke it within the same method.

Reply:
Marking this as resolved as now fieldArrNames is being passed throughout.